model/modelindexer/indexer_test.go

```diff
@@ -127,21 +130,22 @@ func TestModelIndexerAvailableBulkIndexers(t *testing.T) {
 		require.NoError(t, err)
 	}
 	stats := indexer.Stats()
-	stats.BytesTotal = 0
 	// FlushBytes is set arbitrarily low, forcing a flush on each new
 	// event. There should be no available bulk indexers.
```

Comment:
IMO the assertion on BytesTotal distracts from the rest of this test. We're already checking it's working in TestModelIndexer, WDYT about dropping it from this test and go back to setting BytesTotal to zero before comparing?